package.json

```diff
@@ -43,5 +43,8 @@
     "require": [
       "esm"
     ]
+  },
+  "dependencies": {
+    "@typescript-eslint/eslint-plugin-tslint": "^1.6.0"
```

Comment:
We don't want this one, see above. Also, this should be a `devDependency`.

Missing the import and unicorn plugin dependency here. Might not need them at all.

Reply:
Still has `tslint` in `devDependencies`.

Reply:
We need `prettier` to work with `eslint`.